src/test/scala/com/gu/salesforce/messageHandler/MessageHandlerLambdaTest.scala

```diff
@@ -0,0 +1,59 @@
+package com.gu.salesforce.messageHandler
+
+import java.io.{ ByteArrayOutputStream, OutputStream }
+
+import com.amazonaws.services.sqs.model.SendMessageResult
+import com.gu.salesfoce.messageHandler.{ MessageHandler, QueueClient, SqsClient }
+import org.specs2.mutable.Specification
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.util.{ Success, Try }
+import org.specs2.mock.Mockito
+import play.api.libs.json.Json
+
+case class TestLambda(queueClient: QueueClient) extends MessageHandler
+
+class MessageHandlerLambdaTest extends Specification with Mockito {
+  def getTestInputStream(fileName: String) = {
+    getClass.getResourceAsStream(s"/$fileName")
+  }
+
```

Comment:
I commented the unit test since it was taking too much time to make it work. It looks like the automatic parsing using the generated java classes needs some additional setup (possibly logging) and crashes if you run it in a unit test